lowelldevclub/templates/home.html

```diff
@@ -14,20 +14,31 @@
 	<div class="fade-in">
 
 	<div id="particles-js"></div>
-		<nav class="navbar">
-			<a href="/" class="navbar-text">Home</a>
-			<div class="navbar-right">
-				<a href="/about" class="navbar-text" id="">About</a>
+
```

Comment:
navbar should maybe go in base.html

Reply:
It's kind of complicated. I've designed the navbar and footer to go with the particle js background. Maybe to match the theme of our site we can add the particle js in the background and have a container to hold the profile cards and the about content.